src/main/resources/data-repository-openapi.yaml

```diff
@@ -719,6 +719,37 @@ paths:
           schema:
             $ref: '#/definitions/ErrorModel'
 
+  '/api/repository/v1/studies/{id}/paths':
```

Comment:
hmmm. should paths be plural here? i know in rest land the nouns are plural (like studies) but I usually think of that as actual objects and if you want one you follow it with an id. so i would have expected the url to be .../paths/{pathid}. i think if we really wanted to be "rest-y" this would be a POST like querying for an object by a name.

Reply:
Got me. If we extend the GET to do other things, then maybe it should be.
I was trying to be consistent...